irods/pool.py

```diff
@@ -36,6 +36,7 @@ def __init__(self, account, application_name='', connection_refresh_time=-1):
         else:
             self.refresh_connection = False
             self.connection_refresh_time = None
+        logger.debug("In Pool's init(). self.refresh_connection: {}, self.connection_refresh_time: {}".format(self.refresh_connection, self.connection_refresh_time))
```

Comment:
Added a debug statement to print whether irods is configured to refresh connections and if so, what is the connection refresh time.

Reply:
you want to leave this in?   or just helpful during debugging the development?

Reply:
Its helpful for debugging. I can remove it.